lib/components/terms.js

```diff
@@ -64,6 +64,13 @@ export default class Terms extends Component {
     this.terms[uid] = term;
   }
 
+  componentDidMount() {
+    window.addEventListener('contextmenu', () => {
```

Comment:
I didn't know/see there was this event.
In this case, please remove any mention of `right click` notion. We should only speak about context menu.